src/view/ReferralsView.vue

```diff
@@ -1,10 +1,136 @@
 <template>
-    <h1>Referrals view</h1>
+    <section class="referrals">
+        <div class="referrals__header">
+            <greetings-component />
+        </div>
+        <div class="referrals__actions">
+            <button-component
+                class="referrals__make_visit"
+                @click="openModalReferrals = true">
+                Dodaj skierowanie
+                <font-awesome-icon icon="fa-solid fa-plus" />
+            </button-component>
+
+            <teleport to="body">
+                <transition name="modal">
+                    <div
+                        v-if="openModalReferrals"
+                        class="modal">
+                        <modal-component
+                            ref="modalReferrals"
+                            @close="openModalReferrals = false">
+                            <referrals-component />
+                        </modal-component>
+                    </div>
+                </transition>
+            </teleport>
+
+            <button-component
+                class="referrals__make_visit"
+                @click="openModalImportReferrals = true">
+                Importuj skierowanie
+                <font-awesome-icon icon="fa-solid fa-plus" />
+            </button-component>
+
+            <teleport to="body">
+                <transition name="modal">
+                    <div
+                        v-if="openModalImportReferrals"
+                        class="modal">
+                        <modal-component
+                            ref="modalImportReferrals"
+                            @close="openModalImportReferrals = false">
+                            <import-referrals-component />
+                        </modal-component>
+                    </div>
+                </transition>
+            </teleport>
+        </div>
+        <article class="referrals__listOf"></article>
+        <article class="referrals__import"></article>
+        <aside class="referrals__userInfo">
+            <user-sidebar-component />
+        </aside>
+    </section>
 </template>
 
-<script setup></script>
+<script setup>
+    import ButtonComponent from '../components/ButtonComponent.vue';
+    import useUserStore from '../stores/userStore.js';
+    import ModalComponent from '../components/ModalComponent.vue';
+    import GreetingsComponent from '../components/GreetingsComponent.vue';
+    import UserSidebarComponent from '../components/profile/userSidebar/UserSidebarComponent.vue';
+    import ImportReferralsComponent from '../components/referrals/ImportReferralsComponent.vue';
+    import ReferralsComponent from '../components/referrals/ReferralsComponent.vue';
+
+    import { onMounted, ref } from 'vue';
+    import { onClickOutside } from '@vueuse/core';
```

Comment:
as you can see here youve added yourself spaces between imports, declaring vars and functions ;)